api_fhir_r4/converters/patientConverter.py

```diff
@@ -328,12 +329,19 @@ def build_fhir_addresses(cls, fhir_patient, imis_insuree):
 
                 # municipality extension
                 extension = Extension.construct()
-                extension.url = "https://openimis.github.io/openimis_fhir_r4_ig/StructureDefinition/municipality"
+                extension.url = f"{GeneralConfiguration.get_system_base_url()}StructureDefinition-address-municipality"
```

Comment:
Please look on last specification. The URL is StructureDefinition/address-municipality. This is generic URL for FHIR. Please change to comply with IG. The same for other URLs.

Reply:
ok. I will update this and the rest of urls.